src/test/java/net/reini/rabbitmq/cdi/ConsumerHolderTest.java

```diff
@@ -89,6 +91,21 @@ void activateAndDeactivate() throws IOException, TimeoutException {
     verify(channelMock).close();
   }
 
+  @Test
+  void testActivationOrder() throws IOException {
+    sut = new ConsumerHolder(eventConsumerMock, "queue", false, PREFETCH_COUNT,
+        consumerChannelFactoryMock, declarationsListMock, declarerRepositoryMock);
+    Assertions.assertEquals("queue", sut.getQueueName());
+    when(consumerChannelFactoryMock.createChannel()).thenReturn(channelMock);
+    sut.activate();
```

Comment:
In order to have at least one  "assert" you could write:
`assertDoesNotThrow(sut::activate)`

Reply:
fixed